### PR TITLE
Updates to QRAVE Installation Documentation

### PR DESCRIPTION
**Pull request by shelbysawyer**
I added a description of the installation and update process for QRAVE, a video walkthrough, and changed the homepage to include a button for the QRAVE install process.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -5,13 +5,17 @@ title: RAVE AddIn
...
+<a class="button large" href="https://data.riverscapes.xyz">
+        <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
+        &nbsp;&nbsp;Explore WebRAVE in Warehouse/a>
 </div>
 
-![RAVE]({{ site.baseurl}}/assets/images/rave.png)
-
 ## Background
 
 RAVE AddIn is part of the [Riverscapes Consortium's](http://riverscapes.xyz) much larger family of tools for analyzing riverscapes. RAVE itself does not do any analysis, but allows exploring of any [Riverscapes-compliant project](https://riverscapes.github.io/riverscapes-website/Technical_Reference/Documentation_Standards/Riverscapes_Projects/). Rave provides an explorer in which you can navigate the "project tree", and add any layer to map symbolized appropriately.
```
Comment (joewheaton) [inline docs/index.md:21]: Update this text so it is consistent about RAVE (spelling) and not sometimes referring to RAVEAdd In.

**Review by joewheaton — CHANGES_REQUESTED** [review]
This looks pretty good. Please update that paragraph in index.md per my comment and then I can review.

**Comment by shelbysawyer** [comment]
@joewheaton I updated `index.md` to fix typos and have consistent language referring to RAVE.

**Comment by joewheaton** [comment]
Looks great 👍

**Comment by MattReimer** [comment]
I'm reassigning this to @philipbaileynar. It's a pretty easy merge. @joewheaton I'm assuming that you've approved these changes even thoughn your approval still says pending.

File: docs/index.md [hunk 2]
```diff
@@ -5,13 +5,17 @@ title: RAVE AddIn
...
+<a class="button large" href="http://rave.riverscapes.xyz/install.html#qgis-plug-in">
+        <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
+        &nbsp;&nbsp;Installation Instructions for QRAVE PlugIn for QGIS</a>
 <a class="button large" href="https://github.com/Riverscapes/RaveAddIn/releases/latest">
         <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
-        &nbsp;&nbsp;Download RAVE</a>
+        &nbsp;&nbsp;Download ArcRAVE AddIn for ArcGIS 10.X</a>
+<a class="button large" href="https://data.riverscapes.xyz">
+        <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
+        &nbsp;&nbsp;Explore WebRAVE in Warehouse/a>
```
Comment (joewheaton) [inline docs/index.md:16]: open anchor tag with `Warehouse / a>` on 16. Needs to be `Warehouse</a>`. @shelbysawyer this is why website now looks like this:
image

@shelbysawyer are you testing these [MD sites on a local server](https://riverscapes.xyz/Tools/Technical_Reference/Documentation_Standards/WebSites/testing_locally.html)? If you were, you'd see this problem.

**Review by joewheaton — COMMENTED** [review]
The index has an open anchor tag and needs fixing (see my line specific comments).

**Comment by joewheaton** [comment]
> I'm reassigning this to @philipbaileynar. It's a pretty easy merge. @joewheaton I'm assuming that you've approved these changes even thoughn your approval still says pending.

Actually, no... I didn't review until now. see my new review and comments.

File: docs/index.md (hunk 2 above)
Comment (shelbysawyer) [reply docs/index.md:16]: @joewheaton No, I wasn't aware of how to do that. Thanks for the link.

**Pull request merged**